openduck-py/openduck_py/routers/voice.py

```diff
@@ -231,7 +228,6 @@ async def connect_daily(
         session_id=session_id,
         record=False,
         input_audio_format="int16",
-        tts_config=TTSConfig(provider="local", voice_id=voice_id),
```

Comment:
let's keep this parameter and `TTSConfig`. I should be able to change TTS provider at the level of the ResponseAgent rather than globally across the repo.